pbsmetrics/go_metrics.go

```diff
@@ -256,6 +260,14 @@ func makeDeliveryMetrics(registry metrics.Registry, prefix string, bidType openr
 // There is no getBlankAccountMetrics() as all metrics are generated dynamically.
 func (me *Metrics) getAccountMetrics(id string) *accountMetrics {
 	var am *accountMetrics
+	/* type accountMetrics struct {
```

Comment:
Was this supposed to be a temporary note, or do you think we should document the the struct's definition in comments here?

Reply:
Forgot to get rid of this comment. Gone.